src/components/Tile.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Tile = ({ imageSrc, title, paragraphs }) => {
+
+    return (
+        <section>
+            { imageSrc ? <img src={imageSrc} alt="" /> : <h2>{title}</h2> }
+            {paragraphs && paragraphs.length > 0 && (
+                <>
+                    {paragraphs.map((paragraph, index) => (
+                        <p key={index}>{paragraph}</p>
```

Comment:
Je mapt hier over een array van paragraphs, denk ik? Je zou hier je {children} kunnen zetten en dan wordt alles er ook ingezet. Moet je paragraphs invoer in App.js wel een < p > zijn ipv array.